test/e2e/rootless_test.go

```diff
@@ -251,4 +251,16 @@ var _ = Describe("Podman rootless", func() {
 	It("podman rootless rootfs --ipc host", func() {
 		runRootlessHelper([]string{"--ipc", "host"})
 	})
+	It("podman rootless --shm-size", func() {
+		// Check if we can create an user namespace
+		err := exec.Command("unshare", "-r", "echo", "hello").Run()
+		if err != nil {
+			Skip("User namespaces not supported.")
+		}
+		env := os.Environ()
+		cmd := podmanTest.PodmanAsUser([]string{"run", "--shm-size=40m", ALPINE, "grep", "shm", "/proc/self/mountinfo"}, 1000, 0, env)
```

Comment:
this won't be enough to setup the rootless environment.  What we are doing now, through `runRootlessHelper`, since we have no user configured is to setup the environment and use a single mapping inside of the container that runs with `--rootfs`.

Could you add the new test to `runRootlessHelper`?  Otherwise feel free to skip it for now and I can take a look at it.

Reply:
Will try one more time.